client/src/main/java/org/spine3/time/Formats.java

```diff
@@ -0,0 +1,211 @@
+        GregorianCalendar calendar = Calendars.newProlepticGregorianCalendar(timeZone);
+        sdf.setCalendar(calendar);
+        return sdf;
+    }
+
+    /**
+     * Creates a format instance for a format that uses only digits.
+     *
+     * <p>The digit-only format does not require a locale.
+     */
+    private static SimpleDateFormat createDigitOnlyFormat(String dateFormat) {
+        @SuppressWarnings("SimpleDateFormatWithoutLocale") // See Javadoc.
+        final SimpleDateFormat result = new SimpleDateFormat(dateFormat);
+        return result;
+    }
+
+    //
+    // Default time zone formats
+    //----------------------------
+
+    static DateFormat dateTimeFormat() {
+        return dateTimeFormat.get();
+    }
+
+    static DateFormat dateFormat() {
+        return dateFormat.get();
+    }
+
+    static DateFormat timeFormat() {
+        return timeFormat.get();
+    }
+
+    // Zoned formats
+    //---------------------
+
+    static DateFormat dateTimeFormat(ZoneOffset offset) {
+        final TimeZone timeZone = toTimeZone(offset);
+        return createDateTimeFormat(timeZone);
+    }
+
+    private static TimeZone toTimeZone(ZoneOffset offset) {
+        final TimeZone timeZone = ZoneConverter.instance()
+                                               .reverse()
+                                               .convert(offset);
+        checkNotNull(timeZone);
+        return timeZone;
+    }
+
+    static DateFormat dateFormat(ZoneOffset offset) {
+        final TimeZone timeZone = toTimeZone(offset);
+        return createDateFormat(timeZone);
+    }
+
+    static DateFormat timeFormat(ZoneOffset offset) {
+        final TimeZone timeZone = toTimeZone(offset);
+        return createTimeFormat(timeZone);
+    }
+
+    // String generation
+    //----------------------
+
+    /**
+     * Appends the fractional second part of the time to the passed string builder.
+     */
+    static void appendSubSecond(StringBuilder builder, LocalTime time) {
+        final long nanos = LocalTimes.getTotalNanos(time);
+        if (nanos != 0) {
+            builder.append(SUB_SECOND_SEPARATOR);
+            builder.append(formatNanos(nanos));
+        }
+    }
+
+    /** Format the nano part of a timestamp or a duration. */
+    private static String formatNanos(long nanos) {
+        // Determine whether to use 3, 6, or 9 digits for the nano part.
+        if (nanos % NANOS_PER_MILLISECOND == 0) {
+            return format("%1$03d", nanos / NANOS_PER_MILLISECOND);
+        } else if (nanos % NANOS_PER_MICROSECOND == 0) {
+            return format("%1$06d", nanos / NANOS_PER_MICROSECOND);
+        } else {
+            return format("%1$09d", nanos);
+        }
+    }
+
+    /**
+     * Appends the string with zone offset info to the passed string builder.
+     */
+    static void appendZoneOffset(StringBuilder builder, ZoneOffset offset) {
+        if (offset.getAmountSeconds() == 0) {
+            builder.append(UTC_ZONE_SIGN);
```

Comment:
This line (or `OffsetDateTimes.create()` method) seems to be a reason of the failing tests. 

The thing is that in the original `ZoneOffset` the ID is cleared (see `AbstractZonedTimeTest.convert_values_at_current_time_zone()`). So its representation in `now()`has empty "offset":
```
Now is: date { year: 2017 month: APRIL day: 24 } time { hours: 12 minutes: 11 seconds: 18 millis: 197 } offset { } 
```

But in this method we always append `Z` if the "seconds" amount is zero. So in its string representation we get this:
```
String is: 2017-04-24T12:11:18.197Z 
```
Parsed, it looks like this:
```
Parsed is: date { year: 2017 month: APRIL day: 24 } time { hours: 12 minutes: 11 seconds: 18 millis: 197 } offset { id: "UTC" } 
```

We need to make the behaviour consistent. The best fix would be to modify `OffsetDateTimes.create()` to transform the given `ZoneOffset` into `UTC`, if it is empty.